Form-Controls/styles.css

```diff
@@ -0,0 +1,64 @@
+body {
+    background-color: rgb(235, 235, 239);
+}
+.order-form input[type="submit"] {
+  background-color: #d46e42;
+  color: white;
+  padding: 10px 30px;
+  cursor: pointer;
+  margin-top: 10px;
+  margin-bottom: 60px;
+  border: 2px solid #d08666;
+  border-radius: 4px;
+}
+.order-form input[type="submit"]:hover {
+  color: #d08666;
+  background-color: rgb(244, 240, 240);
+  border-color: rgb(220, 208, 208);
+}
+h1, h2{
+  color: #c05326;
+  font-weight: bolder;
+  margin:40px;
+}
+.container {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  gap: 0;
+  margin-left: 60px;
+}
+.order-form label,
+.order-form input,
+.order-form select,
+.order-form input[type="date"],
+.order-form input[type="submit"],
+.order-form input[type="number"] {
+  margin-bottom: 40px;
+  font-weight: bold;
+}
+
+.order-form input,
+.order-form select,
+.order-form input[type="date"],
+.order-form input[type="submit"],
+.order-form input[type="number"] {
+  border: 2px solid #d08666;
+  border-radius: 5px;
+  display: block;
+  margin-top: 10px;
+  width:80%;
```

Comment:
This line here is the root of your email alignment issue. All of your order form elements are 80% of their container, but the email is the only element that takes up a whole line - the others are two elements per line, which makes the 80% a different 80% to all the other elements - confused? Yep, me too. 

To fix this you need to make the width of the email element 90% (I can explain the maths if you want, but it's not important) - use the element id to do this:

```
#customerEmail {
    width: 90%;
}
```